java/scripts/sign_and_bundle_release.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -e
+
+if [ $# -ne 1 ];
+then
+      echo -e "Usage: $0 \e[4mVERSION\e[0m"
+      echo -e "where \e[4mVERSION\e[0m is part of the tag release/java/<version>, ex v1.0.0"
+      exit 1
+fi
+
+
+echo "downloading latest release from github"
+RELEASE_INFO=$(curl -s -H "Accept: application/vnd.github+json" https://api.github.com/repos/loosebazooka/protobuf-specs/releases/tags/release/java/$1)
```

Comment:
s/loosebazooka/sigstore/

Maybe while we're at it make a `RELEASE_REPO` variable

If you need to test against your repo you can use the `: "${RELEASE_REPO:=sigstore/protobuf-specs}"` [trick](https://stackoverflow.com/questions/2013547/assigning-default-values-to-shell-variables-with-a-single-command-in-bash) so you can override it

Reply:
just gonna keep this non-configurable.